### DISCUSSION TITLE
set vpn dns on 8.8.8.8

### DISCUSSION THREAD

**Discussion by siiinaaa** [2dust/v2rayNG#1855] [Ideas]
plz set vpn dns by defult on 8.8.8.8
the 1.1.1.1 dns suspended in iran :|

**Comment by Clash-zone** [comment]
You can simply change the remote dns from settings.

**Reply by hasanrabiei1370** [reply]
g

**Comment by siiinaaa** [comment]
hi iknow that thanks for ur reply
if u can plz put option to turn on MUX multiplexing in V2rayng like v2rayn
best wishes !
